<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupWindow</class>
 <widget class="QMainWindow" name="SignupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign Up</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
		background-color: #fef8ec;
    }
    QWidget#centralForm {
        background-color: rgb(250, 216, 217);
        border-radius: 20px;
    }
    QLabel#titleLabel {
        font-size: 24px;
        font-weight: bold;
        color: #333333;
    }
    QLabel#loginLabel {
        color: #666666;
    }
    QLabel#loginLink {
        color: #0046BE;
        text-decoration: underline;
    }
    QLineEdit {
        padding: 12px;
        border: 1px solid #E5E7EB;
        border-radius: 8px;
        background-color: #FFFFFF;
        font-size: 14px;
        min-height: 20px;
    }
    QPushButton#submitButton {
        background-color: #0046BE;
        color: white;
        border: none;
        border-radius: 8px;
        padding: 8px;
        font-size: 14px;
        font-weight: bold;
        min-height: 30px;
    }
    QCheckBox {
        color: #666666;
        font-size: 13px;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="centralForm" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="formLayout">
         <property name="leftMargin">
          <number>40</number>
         </property>
         <property name="rightMargin">
          <number>40</number>
         </property>
         <item>
          <widget class="QLabel" name="titleLabel">
           <property name="text">
            <string>Sign up</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="loginTextLayout">
           <item>
            <widget class="QLabel" name="loginLabel">
             <property name="text">
              <string>Already have account?</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="loginLink">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Login here</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="nameInput">
           <property name="placeholderText">
            <string>Enter your name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="emailInput">
           <property name="placeholderText">
            <string>Enter your email</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordInput">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Enter your password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="termsCheckbox">
           <property name="text">
            <string>By signing up you agree to receive updates and special offers</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="submitButton">
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="illustrationLabel">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>600</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/Icons/petlogoshop.jpeg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
